README.md

```diff
@@ -1,2 +1,401 @@
-# example-application
-Example out-of-tree application that is also a module
+![Zephyr Project logo](https://github.com/zephyrproject-rtos/zephyr/blob/v1.14-branch/doc/images/Zephyr-Project.png)
+
+
+# Module Overview
+
+The Zephyr Project is a scalable real-time operating system (RTOS) supporting
+multiple hardware architectures, optimized for resource constrained devices,
+and built with security in mind.
+
+The Zephyr build system supports code tree "plug-ins" called *modules*.
+See the [Zephyr Modules (External projects)][5] for a high-level overview
+of the build system module mechanism.
+
+
+## Scope
+
+The Zephyr eXample Application (ZXA) module is supplementary to the Zephyr
+repository.  This provides:
+
+1. A reference implementation of using the Zephyr module mechanism.
+
+1. A repository for holding example code and config files outside of the
+   Zephyr repository, including:
+
+   - `west` manifest
+   - projects/samples/tests
+   - applications/subsystems/drivers/interface declarations
+   - device tree config files
```

Comment:
Devicetree source files

Reply:
> Devicetree source files

Please note 'devicetree', not 'device tree', is part of this change request.